### ISSUE TITLE
Very large db size due to activitylogs in synced instance.

### ISSUE THREAD

**Issue by daitj** [huridocs/uwazi#3348]
We have two instance private and public instance. Private instance syncs few entities with public instance. 

The db size of private instance is around 300mb while private instance is almost 7gb.

This is the details of private instance's activitylogs collection 
```
> db.activitylogs.totalSize()
7261155328
```

Method types count:
```
{
	"DELETE" : 388708,
	"POST" : 1943749,
	"PUT" : 0,
}
```
URL count:
```
{
	"/api/sync" : 2332278,
	"/api/sync/upload" : 97,
	"/api/users/new" : 2,
	"/api/documents/pdfInfo" : 2,
	"/api/import" : 1,
	"/api/remotepublic" : 4,
	"/api/templates" : 11,
	"/api/settings" : 10,
	"/api/files/upload/custom" : 7,
	"/api/pages" : 9,
	"/api/jsonws/invoke" : 39,
	"/api/auth2fa-secret" : 1,
	"/api/auth2fa-enable" : 1,
	"/api/files" : 1
}
```

Should public instance even store activity logs as nothing is user generated content there, it is all from sync request?

Is it possible to disable activity logs, I couldn't find anything in the wiki or in settings page?

Is there any implications if I delete them manually from the mongo shell as a workaround for now?

Normally storage is not big deal but for routine backups and maintainance it is very time and resource consuming to copy and store large amount of data.

Let me know if you need more information?

**Comment by RafaPolit** [comment]
Thanks for the detailed report, and sorry for the late reply.

We detected this ever-growing activity log on synced instances some time ago, and fixed the issue.  If your log is still growing over time, please ensure you have the newest version of Uwazi installed.

Although the issue got fixed, we did not delete the existing activity logs (unfortunately, there was no way to actually know which were genuine user actions and what was our faulty approach at sync).  Since your scenario, indeed, has no use for the activity log on the "public" instance, you can safely delete the contents of the `activitylogs` collection on the target instance.

As a bit of further explanation, private/public scenario is not the main (or only) purpose of the synchronization flow.  It is one of the scenarios we have used this approach for, especially due to the current user permissions status (which is on the process of getting a big revamp).  Still, there are other scenarios which use the sync flow which do require logs to exist, as you could daisy-chain instances syncing A to B to C.

Disabling activity logs could be a useful improvement.  We currently don't have that feature implemented.

So, to sum up:
- activity logs should no longer grow steadily over time, and only reflect actual updates
- in your scenario, it is safe to manually purge the activiylogs via mongo shell, and it is the advised solution

Please, let us know if you have any further issues.
